test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -37,22 +38,17 @@ Describe "Test-ModuleManifest tests" -tags "CI" {
             "ModuleList"="Modules_InvalidModuleListinModuleManifest";
             "TypesToProcess"="Modules_InvalidManifest";
             "FormatsToProcess"="Modules_InvalidManifest";
-            "RootModule"="Modules_InvalidManifest";
+            #"RootModule"="Modules_InvalidManifest";
             "ScriptsToProcess"="Modules_InvalidManifest"
         }
-        foreach ($parameter in $parametersAndExecptions.Keys) {
+        foreach ($parameter in $parametersAndErrors.Keys) {
+            Write-Warning "Testing $parameter"
             $args = @{$parameter = "doesnotexist.psm1"}
             New-ModuleManifest -Path $testModulePath @args
             Test-Path $testModulePath | Should Be $true
-            $errorId = $parametersAndErrors[$parameter]
-
-            try {
-                Test-ModuleManifest -Path $testModulePath -ErrorAction Stop | Should BeOfType System.Management.Automation.PSModuleInfo
-                throw "Test-ModuleManifest did not throw $errorId"         
-            }
-            catch {
-                $_.FullQulaifiedErrorId | Should Match "$errorId"
-            }
+            [string]$errorId = $parametersAndErrors[$parameter] + ",Microsoft.PowerShell.Commands.TestModuleManifestCommand"
+
+            { Test-ModuleManifest -Path $testModulePath -ErrorAction Stop } | ShouldBeErrorId $errorId
             Remove-Item -Recurse -Force $testModulePath
```

Comment:
This is needed as I'm generating a new invalid manifest each iteration of the loop